README.md

```diff
@@ -0,0 +1,86 @@
+# TransformerQAExecutor
+
+**TransformerQAExecutor** wraps a question-answering model from huggingface. Given some questions and paragraphs/contexts, it extracts the relevant answers from the given paragraphs/contexts.
+
+**TransformerQAExexecutor** receives [`Document`s](https://docs.jina.ai/fundamentals/document/) where each `Document`'s `text` is the question, and its `matches`'s texts are the contexts/paragraphs where answers will be extracted from.
```

Comment:
Not sure on this hierarchy.

Shouldn't the main doc be the context and the chunks be questions?

Reply:
Or main doc is context and `.tags['question']` is where we store the question

Reply:
Addressed.